pkg/util/nvidia/paths.go

```diff
@@ -82,14 +82,19 @@ func nvliblist(nvidiaDir string) ([]string, error) {
 // Paths returns list of nvidia libraries and binaries that should
 // be added to mounted into container if it needs NVIDIA GPUs.
 func Paths(nvidiaDir string, envPath string) ([]string, []string, error) {
+	ldConfig, err := exec.LookPath("ldconfig")
+	if err != nil {
+		sylog.Debugf("%v: ldconfig binary not found in %s, will search in %s", err, os.Getenv("PATH"), envPath)
```

Comment:
1. I don't think it is a good idea to print full PATH value here, simply `not found in PATH` is enough
2. It is common to describe error first and then clarify what was wrong. So this should be `sylog.Debugf("could not find ldconfig: %v", err)`.